lib/src/constants/reader.dart

```diff
@@ -122,7 +123,7 @@ abstract class ConstantReader {
 
 class _NullConstant extends ConstantReader {
   @alwaysThrows
-  static dynamic _throw(String expected) {
+  static T _throw<T>(String expected) {
```

Comment:
why not just set `void` as the return type?

Reply:
Because then it makes strong mode sad. The return value is needed since this is used in ` => ` calls

Reply:
ah I see ok